modules/api/src/main/java/org/apache/ignite/table/TupleImpl.java

```diff
@@ -385,10 +384,4 @@ private void readObject(java.io.ObjectInputStream in) throws IOException, ClassN
             colMapping.put(colNames.get(i), i);
         }
     }
-
-    /** {@inheritDoc} */
-    @Override
-    public String toString() {
```

Comment:
Should we preserve some version of "toString"? It is possible to format it manually

Reply:
Agree, we should have a user-friendly `toString` for Tuple.

Reply:
My goal is to avoid using internal utilities in the `api` module at all (perhaps, this idea is pure purism that does not bring valuable benefits, but anyway...).

`IgniteToStringBuilder.toString` is an internal utility that is based on some environment variables to hide sensitive information. I think, that this behavior should be controlled via configuration instead of environment variables, so it will be hard to move this utility into the `api` module in the future. However, I agree that we should have a user-friendly method to print tuples.

I see the following alternatives:
 - move `IgniteToStringBuilder` utility to the `api` module (including annotations like `IgniteToStringExclude`, `IgniteToStringInclude`, `IgniteToStringOrder`, and so on). The behavior is still controlled by env variables (perhaps, it is possible to offer a wiser approach).
 - add simple implementation `toString` to the `TupleImpl` class and improve `IgniteToStringBuilder` implementation to handle such cases taking into account sensitive information.
 - move `TupleImpl` implementation into an internal package and provide new methods to create instances via `Ignite` instance.

For now, I would add a simple implementation `toString` to the `TupleImpl` and after that, it would be nice to discuss the alternatives.